101-servicebus-create-namespace-geo-recoveryconfiguration/README.md

```diff
@@ -0,0 +1,11 @@
+# 101-servicebus-create-namespace-geo-recoveryconfiguration
```

Comment:
We need a proper readme here - see the contribution guide.  If nothing else add the text from metadata.json here.

Reply:
updated the readme